### ISSUE TITLE
LDAP sync doesn't work? no logging

### ISSUE THREAD

**Issue by zyxep** [h44z/wg-portal#36]
First of all, thanks for a great tool so far there is a lot of potential in this.

I'm trying to hook it up to our FreeIPA server but it doesn't seem to sync users over.
i'ved tried to enable trace log or even debug, no information on that.
So I don't really know where it's stuck, if it's the connection, DN, filters (i use default in this case).

**Comment by h44z** [comment]
Hi, if ldap sync is working, a trace log message ("syncing ldap users...") should be visible. If that is not the case, you probably forgot to enable the ldap feature all together. Is the config option LDAP_ENABLED set to true?

**Comment by zyxep** [comment]
`ldapEnabled` is enabled in my .yml file.

are there any debugging for the connection it self?

**Comment by zyxep** [comment]
I only see `TRAC[2021-07-20 07:03:13] syncing ldap users to database...` in the output.
I don't run it in Docker, but just on the host it self :) with the golang app.

**Comment by h44z** [comment]
Only error messages are logged for the sync. If there is no further message I assume that the ldap filter results in an empty user array. The default ldap filter was tested for active directory only. Maybe you can validate the filter filter string?

**Comment by zyxep** [comment]
I got them to sync.
You were right, it was the filters.

I ended up with:
```
  loginFilter: (&(objectClass=organizationalPerson)(mail={{login_identifier}})(!(nsAccountLock=TRUE)))
  syncFilter: (&(objectClass=organizationalPerson)(mail=*)(!(nsAccountLock=TRUE)))
```

With those filtes, it looks at only active users, @h44z  maybe a feature request here: on the trace log or maybe even debug log, have it saying that it didn't find any users. then it would probably have given me insights in my filters was wrong.

disclaimer for others: **THIS IS ONLY TESTED FOR FreeIPA**
